src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/Talisman.java

```diff
@@ -206,7 +208,8 @@ private static void activateTalisman(Event e, Player p, Inventory inv, Talisman
         consumeItem(inv, talisman, talismanItem);
         applyTalismanEffects(p, talisman);
```

Comment:
Shouldn’t the event be fired before the effects are applied? Then cancelling it will actually have meaning then?

Reply:
yup, added this in the cancellation logic